scripts/mkcloud

```diff
@@ -407,18 +407,23 @@ function setupadmin
     onadmin write_cloud_info
     echo "you can now proceed with installing crowbar"
     # prevent jumbo frames from going out
+    local IPTABLES_WAITOPT=
+    if $($sudo iptables --wait --list INPUT > /dev/null); then
```

Comment:
You could even drop the `sudo` if you call it with path such as

~~~
/usr/sbin/iptables --help [...]
~~~